### ISSUE TITLE
Function sanityIntegration is not the default export anymore

### ISSUE THREAD

**Issue by olach** [sanity-io/sanity-astro#138]
**Describe the bug**

This Astro integration is not installed correctly when following the suggested installation instructions.

**To Reproduce**

1. Run `npm create astro@latest`
2. Run `npx astro add @sanity/astro`

The following is shown in the terminal:

```
Astro will make the following changes to your config file:

 ╭ astro.config.mjs ─────────────────────────────╮
 │ import { defineConfig } from 'astro/config';  │
 │                                               │
 │ import sanity from "@sanity/astro";           │
 │                                               │
 │ // https://astro.build/config                 │
 │ export default defineConfig({                 │
 │   integrations: [sanity()]                    │
 │ });                                           │
 ╰───────────────────────────────────────────────╯
```

3. Run `npm run build`
4. Error occurs: `[astro] Unable to load your Astro config`

**Workaround**

```
// Original:
// import sanity from "@sanity/astro"
// Fix:
import { sanityIntegration as sanity } from "@sanity/astro"
```

**Source of error**

As per [this comment](https://github.com/sanity-io/sanity-astro/issues/92#issuecomment-1792958620) the error is because it looks like [sanityIntegration](https://github.com/sanity-io/sanity-astro/blob/main/packages/sanity-astro/src/index.ts#L14) is not the default export anymore.

**Suggested fix**

Either fix so that `sanityIntegration` is the default export, or fix so the automatic configuration of the Astro config file is edited correctly.

There are another issues (#92) and PR (#111) that also talks about this issue that needs to be fixed.

**Comment by narration-sd** [comment]
Ola, good catch here.

`sanityIntegration` is indeed the export (it isn't a default export) of the package, has been.

I don't find a way to configure what the `astro add` command does in the package code, so I'm imagining it gets set up when the integration is noted to Astro, using the Submit form on the top left of [this page](https://astro.build/integrations/).

Someone at Sanity development should do that, would seem, so I'll put a note in the current PR about it, referencing your issue here.

**Comment by kmelve** [comment]
Should be solved in #149 – thanks for reporting! (Astro requires a default export for `npx astro add` installation)
